### PR TITLE
Initial commit with the code pulled out of our main app.

### PR DESCRIPTION
**Pull request by scott-w**
@kennydude I made a minor change to get rid of unused imports and other PEP8 stuff - can you run over this and make sure it all looks good.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Django==1.9.6
+argparse==1.2.1
+djangorestframework==3.3.3
+wsgiref==0.1.2
```
Comment (kennydude) [inline requirements.txt:4]: Will this file require these versions by anyone who includes the library?

Comment (scott-w) [inline requirements.txt:4]: No, my intention is that this is just used for testing.

Comment (kennydude) [inline requirements.txt:4]: Cool :+1:

**Comment by kennydude** [comment]
:+1: Good to go

**Pull request merged**